schemas/nsgv_network_port_vlans.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "urn:nuage-metroae:nsgv-network-port-vlans",
+    "title": "NSGv Network Port VLANs",
+    "description": "Specify NSGvs network port VLAN configuration.",
+    "type": "array",
+    "widget": "form",
+    "listName":"nsgv_network_port_vlans",
+    "items": {
+        "widget": "item",
+        "type": "object",
+        "title": "NSGv",
+        "additionalProperties": false,
+        "properties": {
+            "name": {
+                "type": "string",
+                "title": "NSGv Network Port VLAN Name",
+                "description": "VLAN name of the NSGv network port",
+                "default": "",
+                "propertyOrder": 10,
+                "sectionBegin": "Network ports"
```

Comment:
Looks like the Section Begin here and in the next one are the same. Every section begin should have the same section end to end the section. You can run src/generate_all_from_schema.sh to generate new examples everytime schema changes are made. The script does not run on windows so needs to be run on a Linux machine. Else, there is also a jenkins job that can generate the examples.

Reply:
bad copy/paste :)
Fixed.

Reply:
We still need to regenerate examples so that the excel sheet examples have the correct format.

Reply:
I added the example files. Lemme know if that's how it's usually done.

Reply:
We automate the example generation using src/generate_all_from_schema.sh . That script can be run only from a linux machine unfortunately. That will regenerate all examples.